frameworks/Java/jooby/src/main/java/com/techempower/App.java

```diff
@@ -44,57 +54,145 @@
 
   static final String HELLO_WORLD = "Hello, World!";
 
+  static final ByteBuffer HELLO_WORLD_BUFFER = ByteBuffer
+      .wrap(HELLO_WORLD.getBytes(StandardCharsets.UTF_8));
+
+  @Inject
+  private DataSource ds;
+
   {
-    /** templates via rocker. */
+    /** Template engine: */
     use(new Rockerby());
 
-    /** json via jackson . */
+    /** JSON: */
     ObjectMapper mapper = new ObjectMapper();
     use(new Jackson(mapper));
 
-    /** database via requery. */
+    /** Database: */
     use(new Jdbc());
-    use(new Requery(Models.DEFAULT));
-
-    get("/plaintext", () -> result(HELLO_WORLD, text))
-        .renderer("text");
-
-    get("/json", () -> result(mapper.createObjectNode().put("message", HELLO_WORLD), json))
-        .renderer("json");
-
-    get("/db", req -> {
-      int id = ThreadLocalRandom.current().nextInt(DB_ROWS) + 1;
-      EntityStore<Persistable, World> store = require(EntityStore.class);
-      World world = store.select(World.class)
-          .where(World.ID.eq(id))
-          .get()
-          .first();
-      return result(world, json);
+
+    get("/plaintext", (req, rsp) -> {
+      rsp.send(result(HELLO_WORLD_BUFFER, plain));
+    }).renderer("byteBuffer");
+
+    get("/json", (req, rsp) -> {
+      rsp.send(result(mapper.createObjectNode().put("message", HELLO_WORLD), json));
```

Comment:
@jknack Oh, thank you! Feel free to make the exact same change there if you'd like, and I'll add it to that issue for tracking.